pubsub/azurepubsub/azurepubsub.go

```diff
@@ -18,10 +18,13 @@
 //
 // URLs
 //
-// For pubsub.OpenTopic/Subscription URLs, azurepubsub registers for the scheme
-// "azuresb". The Service Bus Connection String defaults to the environment
-// variable "SERVICEBUS_CONNECTION_STRING". For details on the format of the
-// URL, see URLOpener.
+// For pubsub.OpenTopic and pubsub.OpenSubscription, azurepubsub registers
+// for the scheme "azuresb".
+// The default URL opener will use a Service Bus Connection String based on
```

Comment:
Probably in another PR, I think the package should be renamed to `servicebus` to be consistent.

Reply:
You are probably right. I'm not sure it's worth it at this point though. `awspubsub` should probably be something more like `snssqs` too.

On the other hand, I think it's nice to have `azure` in there to make it more recognizable.

Let's discuss at standup tomorrow.